dietpi/dietpi-software

```diff
@@ -6718,6 +6725,111 @@ _EOF_
 
 		fi
 
+		software_id=204 # Navidrome
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 ))
+		then
+
+			Banner_Installing
+
+			# ARMv6
+			local arch='armv6'
+
+			# ARMv7
+			if (( $G_HW_ARCH == 2 )); then
+				arch='armv7'
+
+			# ARMv8
+			elif (( $G_HW_ARCH == 3 )); then
+				arch='arm64'
+
+			# x86_64
+			elif (( $G_HW_ARCH == 10 )); then
+				arch='x86_64'
+
+			fi
+			
+			local fallback_url="https://github.com/navidrome/navidrome/releases/download/v0.47.5/navidrome_0.47.5_Linux_$arch.tar.gz"
+			Download_Install "$(curl -sSfL 'https://api.github.com/repos/navidrome/navidrome/releases/latest' | mawk -F\" "/\"browser_download_url\": .*\/navidrome_[0-9.]*_Linux_$arch\.tar\.gz\"/{print \$4}")" /opt/navidrome/
+
+			# Data dir
+			[[ -d '/mnt/dietpi_userdata/navidrome' ]] || G_EXEC mkdir /mnt/dietpi_userdata/navidrome
+
+			# User
+			Create_User -g dietpi -d /mnt/dietpi_userdata/navidrome navidrome
+
+			# Service: https://www.navidrome.org/docs/installation/linux/#create-configuration-file
+			cat << '_EOF_' > /mnt/dietpi_userdata/navidrome/navidrome.toml
+DataFolder = '/mnt/dietpi_userdata/navidrome/data'
+MusicFolder = '/mnt/dietpi_userdata/Music'
+ImageCacheSize = '100MiB'
+
+LogLevel = 'info'
```

Comment:
For testing, info is nice, but for production I suggest to reduce the log level:
```suggestion
LogLevel = 'warn'
```